execution/gethexec/blockchain.go

```diff
@@ -65,6 +69,18 @@ var DefaultCachingConfig = CachingConfig{
 	SnapshotRestoreGasLimit:            300_000_000_000,
 	MaxNumberOfBlocksToSkipStateSaving: 0,
 	MaxAmountOfGasToSkipStateSaving:    0,
+	StateScheme:                        rawdb.HashScheme,
+	StateHistory:                       90000,
```

Comment:
It would be great to have some input about a good default value for StateHistory :)

Reply:
let's bring it up for a broader discussion.

Reply:
As discussed, by now I set StateHistory to a fixed value.
I created a [ticket](https://linear.app/offchain-labs/issue/NIT-2597/set-pathdbs-statehistory-to-the-latest-safe-block-instead-of-being-a) to change the strategy here, to set the StateHistory to the latest safe block instead,